src/implementations/twig/components/news-ticker/news-ticker.html.twig

```diff
@@ -93,56 +102,60 @@
     <span class="ecl-news-ticker__counter--current">1</span> {{ _counter_label }} {{_items|length}}
   </div>
   <div class="ecl-news-ticker__controls">
-    {% include '@ecl/button/button.html.twig' with {
-      type: 'button',
-      variant: 'primary',
-      icon: {
-        path: _icon_path,
-        name: 'corner-arrow',
-        size: 'm',
-        transform: 'rotate-270',
-      },
-      extra_classes: 'ecl-news-ticker__prev',
-      extra_attributes: [
-        {name: 'data-ecl-news-ticker-prev'},
-        {name: 'aria-controls', value: _aria_controls},
-        {name: 'tabindex', value: '-1'},
-        {name: 'data-controls', value: 'prev'},
-      ],
-    } only %}
+    <button 
```

Comment:
Isn't it worth adding a parameter to the button template, to have a label, but hidden?
Something like hide_label: true/false